Zilon.Core/Zilon.Core/Persons/HumanPerson.cs

```diff
@@ -58,6 +58,10 @@ public class HumanPerson : IPerson
 
         public IPlayerEventLogService PlayerEventLogService { get; set; }
 
+        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310821)